### PR TITLE
Update CypherTypes in procedure/function descriptions

### PR DESCRIPTION
**Pull request by gem-neo4j**
We are updating the descriptions in the output for SHOW FUNCTIONS and SHOW PROCEDURES to match the defaults of the new Cypher Types

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/functions/index.adoc [hunk 1]
```diff
@@ -475,211 +475,211 @@ These functions are used to manipulate strings or to create a string representat
...
+`DURATION` values of the xref::values-and-types/temporal.adoc[temporal types] can be created manipulated using the following functions:
 
 [options="header"]
 |===
 | Function | Signature | Description
 
 1.1+| xref::functions/temporal/duration.adoc#functions-duration[`duration()`]
 | `duration(input :: ANY?) :: (DURATION?)`
-| Construct a Duration value.
+| Construct a `DURATION` value.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/index.adoc]:
```suggestion
| Constructs a `DURATION` value.
```

File: modules/ROOT/pages/functions/list.adoc [hunk 2]
```diff
@@ -232,8 +232,8 @@ A list containing all the nodes in the path `p` is returned.
 [[functions-range]]
 == range()
 
-`range()` returns a list comprising all integer values within a range bounded by a start value `start` and end value `end`, where the difference `step` between any two consecutive values is constant; i.e. an arithmetic progression.
-To create ranges with decreasing integer values, use a negative value `step`.
+`range()` returns a list comprising all `INTEGER` values within a range bounded by a start value `start` and end value `end`, where the difference `step` between any two consecutive values is constant; i.e. an arithmetic progression.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/list.adoc]:
```suggestion
`range()` returns a list comprising all `INTEGER` values within a range bounded by a `start` value and an `end` value, where the difference `step` between any two consecutive values is constant; i.e. an arithmetic progression.
```

File: modules/ROOT/pages/functions/list.adoc [hunk 3]
```diff
@@ -36,7 +36,7 @@ CREATE
 [[functions-keys]]
 == keys()
 
-`keys` returns a list containing the string representations for all the property names of a node, relationship, or map.
+`keys` returns a list containing the string representations for all the property names of a `NODE`, `RELATIONSHIP`, or `MAP`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/list.adoc]: This comes up more below as well, so will just ask here.
Should this not be "...returns a `LIST< >`...?

File: modules/ROOT/pages/functions/list.adoc [hunk 4]
```diff
@@ -587,7 +587,7 @@ toBooleanList(list)
 |===
 
 | Any `null` element in `list` is preserved.
-| Any boolean value in `list` is preserved.
+| Any `BOOLEAN` value in `list` is preserved.
 | If the `list` is `null`, `null` will be returned.
 | If the `list` is not a list, an error will be returned.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/list.adoc]: Should these be `LIST< >`?

File: modules/ROOT/pages/functions/list.adoc [hunk 5]
```diff
@@ -622,7 +622,7 @@ toBooleanList(['a string', true, 'false', null, ['A','B']]) as mixedList
 [[functions-tofloatlist]]
 == toFloatList()
 
-`toFloatList()` converts a list of values and returns a list of floating point values.
+`toFloatList()` converts a list of values and returns a `LIST<FLOAT>`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/list.adoc]: See more similar examples below.

```suggestion
`toFloatList()` converts a `LIST<ANY>` and returns a `LIST<FLOAT>`.
```

File: modules/ROOT/pages/functions/list.adoc [hunk 6]
```diff
@@ -554,7 +554,7 @@ The property named `array` and a list comprising all but the first element of th
 [[functions-tobooleanlist]]
 == toBooleanList()
 
-`toBooleanList()` converts a list of values and returns a list of boolean values.
+`toBooleanList()` converts a list of values and returns a `LIST<BOOLEAN>`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/list.adoc]:
```suggestion
`toBooleanList()` converts a `LIST<ANY>` and returns a `LIST<BOOLEAN>`.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 7]
```diff
@@ -1,11 +1,11 @@
-:description: Cypher provides functions allowing for the creation and manipulation of values for a _Duration_ temporal type.
+:description: Cypher provides functions allowing for the creation and manipulation of values for a `DURATION` temporal type.
 
 [[query-functions-temporal-duration]]
 = Temporal functions - duration
 
 [abstract]
 --
-Cypher provides functions allowing for the creation and manipulation of values for a _Duration_ temporal type.
+Cypher provides functions allowing for the creation and manipulation of values for a `DURATION` temporal type.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]: Not part of your PR but that definition is a bit convoluted :) 

```suggestion
Cypher provides functions allowing for the creation and manipulation of temporal `DURATION` values.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 8]
```diff
@@ -256,18 +256,18 @@ duration.between(instant1, instant2)
 | Name | Description
 
 | `instant1`
-| An expression returning any temporal instant type (_Date_ etc) that represents the starting instant.
+| An expression returning any temporal instant type (`DATE` etc) that represents the starting instant.
 
 | `instant2`
-| An expression returning any temporal instant type (_Date_ etc) that represents the ending instant.
+| An expression returning any temporal instant type (`DATE` etc) that represents the ending instant.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]:
```suggestion
| An expression returning any temporal instant type (`DATE` etc.) that represents the ending instant.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 9]
```diff
@@ -256,18 +256,18 @@ duration.between(instant1, instant2)
 | Name | Description
 
 | `instant1`
-| An expression returning any temporal instant type (_Date_ etc) that represents the starting instant.
+| An expression returning any temporal instant type (`DATE` etc) that represents the starting instant.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]:
```suggestion
| An expression returning any temporal instant type (`DATE` etc.) that represents the starting instant.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 10]
```diff
@@ -420,18 +420,18 @@ duration.inDays(instant1, instant2)
...
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]:
```suggestion
| An expression returning any temporal instant type (`DATE` etc.) that represents the starting instant.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 11]
```diff
@@ -420,18 +420,18 @@ duration.inDays(instant1, instant2)
...
 
 | `instant2`
-| An expression returning any temporal instant type (_Date_ etc) that represents the ending instant.
+| An expression returning any temporal instant type (`DATE` etc) that represents the ending instant.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]:
```suggestion
| An expression returning any temporal instant type (`DATE` etc.) that represents the ending instant.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 12]
```diff
@@ -501,18 +501,18 @@ duration.inSeconds(instant1, instant2)
 | Name | Description
 
 | `instant1`
-| An expression returning any temporal instant type (_Date_ etc) that represents the starting instant.
+| An expression returning any temporal instant type (`DATE` etc) that represents the starting instant.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]:
```suggestion
| An expression returning any temporal instant type (`DATE` etc.) that represents the starting instant.
```

File: modules/ROOT/pages/functions/temporal/duration.adoc [hunk 13]
```diff
@@ -501,18 +501,18 @@ duration.inSeconds(instant1, instant2)
...
 
 | `instant2`
-| An expression returning any temporal instant type (_Date_ etc) that represents the ending instant.
+| An expression returning any temporal instant type (`DATE` etc) that represents the ending instant.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/functions/temporal/duration.adoc]:
```suggestion
| An expression returning any temporal instant type (`DATE` etc.) that represents the ending instant.
```

**Review by JPryce-Aklundh — COMMENTED** [review]
It looks really good overall 👍 , but I have some questions/possible suggestions  for the list values page.

**Review by JPryce-Aklundh — APPROVED** [review]

**Pull request merged**